addons/crm/tests/test_crm_lead.py

```diff
@@ -81,3 +82,25 @@ def test_mailgateway(self):
         new_lead.handle_partner_assignment(create_missing=True)
         self.assertEqual(new_lead.partner_id.email, 'unknown.sender@test.example.com')
         self.assertEqual(new_lead.partner_id.team_id, self.sales_team_1)
+
+    @mute_logger('odoo.addons.phone_validation.tools.phone_validation')
+    def test_field_email_and_phone(self):
```

Comment:
Maybe improve tests to use real (fake) numbers, using Be country . That way we could check that changing lead number updates its partner phone_sanitized number, making a more complete test :) .

Reply:
@tde-banana-odoo I can't test it in CRM itself, because it depends on `_sms_get_number_fields` which is defined in `crm_sms` (actually there's no test in the module `crm_sms`, so is it relevant to add a new file just for  that ?)

Reply:
(and when I said that the phone was sanitized on the partner, it's because an onchange, `phone_sanitized` can not be used because it can be the mobile or the phone number)